### DISCUSSION TITLE
I want only Breadcrumb and Dark Mode feature

### DISCUSSION THREAD

**Discussion by ramesh-dada** [vuepress-theme-hope/vuepress-theme-hope#844] [Q&A] [answered]
Hello @Mister-Hope I see this theme is amazing, But I want only dark mode and Breadcrumb feature, So is their any plugin available for it?? as I am not a coder i dont have a that knowledge about this things, Any help will be appreciated.

**Comment by Mister-Hope** [comment, marked as answer]
You are free to disable any features you don't want

**Reply by ramesh-dada** [reply]
Hello can you tell me how to disable features (Sorry if I asked very silly question)

**Reply by Mister-Hope** [reply]
Please just read the docs. the docs is containing everything you want, and we use algolia to provide whole site searching.

---

But, as you want:

Here is a demo: [Source](https://github.com/vuepress-theme-hope/vuepress-theme-hope/tree/v1/demo/) [Preview](https://vuepress-theme-hope-demo.mrhope.site/)

And in [this page](https://vuepress-theme-hope-demo.mrhope.site/guide/disable/) you can see that nearly all features are disabled.

You can read the [source code](https://github.com/vuepress-theme-hope/vuepress-theme-hope/blob/v1/demo/src/guide/disable.md) and understand that part in docs.

**Reply by Mister-Hope** [reply]
Just another reminder, `themeConfig` in config.js is the global config. And you should specific certain pages config  in their frontmatter.

You should config similar keys in `themeConfig` which you found in frontmatter if you want them disabled globally.
